controllers/create.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Can we rename the file to manager.go ? It's mostly related to handling manager, ryt?

Reply:
yes that would be fine. This is where the manager is created and all reconcilers are also added.

Reply:
Done